src/pybind/mgr/mgr_module.py

```diff
@@ -103,6 +103,17 @@ def wait(self):
         return self.r, self.outb, self.outs
 
 
+class HandleCommandResult(namedtuple('HandleCommandResult', ['retval', 'stdout', 'stderr'])):
```

Comment:
What's the advantage of using a subclass rather than just a namedtuple as in HandleCommandResult = namedtuple('HandleCommandResult', 'retval stdout stderr')?  Default args?

Reply:
[Default args](https://stackoverflow.com/a/18348004/3185053). Let's make Python 3.7 mandatory as soon as possible :-)

Alternatively, we could also use [`typing.NamedTuple`](https://docs.python.org/3/library/typing.html#typing.NamedTuple), which is already available in Python 3.6 :-)

Reply:
fair enough.